ena-linkage/main.py

```diff
@@ -131,21 +105,25 @@ def run_additional_checks(response_json: Dict, specimen_data: Dict) -> bool:
     :return: Returns whether it passes for the checks (True) or not (False)
     """
     is_valid = False
-    if specimen_data.get('occurrences')[0].get('dwc:eventDate') and response_json.get('collection_date'):
-        if specimen_data.get('occurrences')[0].get('dwc:eventDate') == response_json.get('collection_date'):
+    if specimen_data.get(HAS_EVENT)[0].get(
+            'dwc:eventDate') and response_json.get('collection_date'):
+        if specimen_data.get(HAS_EVENT)[0].get(
+                'dwc:eventDate') == response_json.get('collection_date'):
             is_valid = True
         else:
             logging.info(
                 f'Event date {response_json.get("collection_date")} does not match for specimen: '
-                f'{specimen_data.get("occurrences")[0].get("dwc:eventDate")}')
+                f'{specimen_data.get(HAS_EVENT)[0].get("dwc:eventDate")}')
```

Comment:
Why is HAS_EVENT in capitals?

Reply:
line 11, global variable declared. 
In java we put these kinds of variables in caps...

Reply:
Ok I can see why, its not a common practice in Python I think, but please adhere to your own coding standard if that works (at least it attracts ;))